php-plugin/src/main/java/fr/greencodeinitiative/php/checks/UseOfMethodsForBasicOperations.java

```diff
@@ -25,7 +25,7 @@
         tags = {"bug", "eco-design", "ecocode"})
 public class UseOfMethodsForBasicOperations extends PHPSubscriptionCheck {
 
-    public static final String RULE_KEY = "D2";
```

Comment:
why `EC22` instead of `EC2` ?
same feedback for files `EC22.html` and `EC22.json`

Reply:
As @ganncamp's [advised](https://community.sonarsource.com/t/new-plugin-ecocode-requesting-inclusion-in-sonarqube-marketplace/85398/10):
> So there’s no need to differentiate rule key prefixes across languages.

So the unicity of the keys is valid for across all the plugins.
`EC2` is already defined in java-plugin as:
> If we are using too many conditional if-else statements it will impact performance since JVM will have to compare the conditions. We can think of using a switch statement instead of multiple if-else if possible. Switch statement has a performance advantage over if – else.

`EC22` is different:
> Use of methods for basic operations

This is why I can't use `EC2` for this PHP rule.